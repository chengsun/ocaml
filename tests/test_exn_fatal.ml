exception Ouch

let () = raise Ouch

let () = Printf.printf "goodbye\n"
