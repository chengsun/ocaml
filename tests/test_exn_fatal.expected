Uncaught OCaml exception: Test.Ouch
